neuralspace/voice_ai.py

```diff
@@ -270,6 +270,36 @@ def poll_until_complete(self, job_id: str, poll_schedule: Optional[List[float]]
         return result
 
 
+    def ama(self, job_id: str, prompts: List[str]) -> Dict[str, Any]:
+        """
+        Create and send a request for an AMA job.
+
+        Parameters
+        ----------
+        job_id: str
+            The job ID for the transcript on which to run AMA.
+        prompts: List[str]
+            List of prompts for the AMA.
+
+        Returns
+        -------
+        result: dict
+            The response from the server.
+        """
+        url = f'{K.FULL_AMA_URL.rstrip("/")}'
+        hdrs = self._create_headers()
+        hdrs['Content-Type'] = 'application/json'
+        data = json.dumps({
+            "jobId": job_id,
+            "prompts": prompts
+        })
+
+        sess = self._get_session()
+        r = sess.post(url, headers=hdrs, data=data)
```

Comment:
Hello, for sending JSON payload, we can omit setting the content-type header manually; omit the json.dumps; and just pass the `data` dict as `sess.post(..., json=data)`